src/lib/index.js

```diff
@@ -1005,14 +1021,15 @@ var maximumNumberOfHoverTemplateWarnings = 10;
  *  Lib.hovertemplateString('name: %{trace[0].name}', {trace: [{name: 'asdf'}]}) --> 'name: asdf'
  *  Lib.hovertemplateString('price: %{y:$.2f}', {y: 1}) --> 'price: $1.00'
  *
- * @param {obj}     d3 locale
  * @param {string}  input string containing %{...:...} template strings
  * @param {obj}     data object containing fallback text when no formatting is specified, ex.: {yLabel: 'formattedYValue'}
+ * @param {obj}     d3 locale
  * @param {obj}     data objects containing substitution values
  *
  * @return {string} templated string
  */
-lib.hovertemplateString = function(string, labels, d3locale) {
+lib.templateFormatString = function(string, labels, d3locale) {
```

Comment:
Very nice abstraction!

Is `Lib.templateFormatString` called directly anywhere outside `src/lib/index.js`? If not, we don't have to export it on `Lib`?

Reply:
Done in ee0a309ed478c121b6693c5833d45cf1c9e1f1ec